recipes-devtools/sbsigntool/sbsigntool/disable-man-page-creation.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/docs/Makefile.am b/docs/Makefile.am
+index 1b5a588..6918dd8 100644
+--- a/docs/Makefile.am
++++ b/docs/Makefile.am
+@@ -1,8 +1,4 @@
+ 
+-man1_MANS = sbsign.1 sbverify.1 sbattach.1 sbvarsign.1 sbsiglist.1 \
```

Comment:
Typically, there is a configuration switch to disable the doc creation. I assume there is no such switch here? That would be the proper way to fix it upstream.

Reply:
@macpijan no, the switch is not present

Reply:
Please add to the list for potential contribution. It should be fairly easy to add such switch